client/pages/report-list/list.css.styl

```diff
@@ -4,13 +4,34 @@
     margin-top 15px
     padding 15px
 
+    .select-status
+      align-items center
+      flex-wrap wrap
+      display flex
+
+      .select-label
+        font-weight bold
```

Comment:
Any reason why this long sentence should be in bold style? I think it looks much better without it.